frontend/src/routes/candidate/(protected)/+page.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import {appName} from '$candidate/placeholder.json';
```

Comment:
This can be removed and the title fetched from the translations. Or you can fix it when this page is properly implemented.